pkg/api/api.go

```diff
@@ -335,6 +335,7 @@ func (hs *HttpServer) registerRoutes() {
 		adminRoute.Get("/users/:id/quotas", wrap(GetUserQuotas))
 		adminRoute.Put("/users/:id/quotas/:target", bind(m.UpdateUserQuotaCmd{}), wrap(UpdateUserQuota))
 		adminRoute.Get("/stats", AdminGetStats)
+		adminRoute.Post("/:dashboardId/pause-dashboard-alerts", bind(dtos.PauseDashboardAlertsCommand{}), wrap(PauseDashboardAlerts))
```

Comment:
I don't think this should be under `/admin`. We already have a route there but I don't it should be there. 
I think it's better to have it under `/api/alerts` https://github.com/grafana/grafana/pull/10622/files#diff-13d216b12482e706601d65bcc87deb38R293

maybe
POST `/api/alerts/pause/by-dashboard/:dashboardId`

not :100: sure yet...

Reply:
And then change the others to
`/api/alerts/pause/by-id/:alertId`
`/api/alerts/pause/all`
?
I found it confusing that `pause-all-alerts` was under the admin route and not the alerts route.

Reply:
I don't want to change the `pause-all-alerts` path since that would be a breaking change for everyone who wrote scripts to do that. But I think we could have two paths do the same thing.